### PR TITLE
Simplified flows v3.4

### PR DESCRIPTION
**Pull request by neasaw**
Readme and Release Documentation for GBM

### CODE REVIEW

**Comment by neasaw** [comment]
GBM Read.me =>https://github.com/CiscoDevNet/webexcc-digital-channels/blob/e48712c14c107f2afcdb1e84ace548116d19022a/Webex%20Connect%20Flows/v3.4/Template/Media%20Specific%20Workflows/Google%20Business%20Messages/README.md

Media Specific WorkFlows =>https://github.com/CiscoDevNet/webexcc-digital-channels/blob/e48712c14c107f2afcdb1e84ace548116d19022a/Webex%20Connect%20Flows/v3.4/Template/Media%20Specific%20Workflows/README.md

Version History =>https://github.com/CiscoDevNet/webexcc-digital-channels/blob/8291f8bcc845cebaa87dd849d73db8262349d406/Webex%20Connect%20Flows/Version%20History.md

V3.3 -> v3.4 =>https://github.com/CiscoDevNet/webexcc-digital-channels/blob/629fd96a95ebfca063d04984aff0eab8eb575f6f/Webex%20Connect%20Flows/How%20to%20manually%20upgrade%20from%20v3.3%20to%20v3.4%20workflows.MD

File: Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+# How to manually upgrade from v3.3 to v3.4 workflows
+
+In case you have made custom changes to your inbound flows, you can manually upgrade your existing v3.3 flows to v3.4. This ensures that the custom changes are retained and don't have to be repeated. Follow the below steps for the same:
+
+## For Google Business Messages 
+- Two new flows have been added in [Google Business Messages](v3.4/Template/Media%20Specific%20Workflows/Google%20Business%20Messages) as references to create flows with Google Business Message Channel 
+
+## For all other flows
+- There are no changes made to other flows they have been moved from v3.3 to v3.4 in pristine condition
```
Comment (siramana) [inline Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD]: We can keep only "There are no changes made to other flows"

File: Webex Connect Flows/Version History.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+##v3.3
+####Features released:
+- Apple Business Messages
+    - Support for Apple Business Messages as a channel
+
+##v3.2
+####Features released:
+- Option to set contact priority while queueing contacts
+- Capability to deliver system messages such as survey links and survey messages to customers post contact handling
```
Comment (siramana) [inline Webex Connect Flows/Version History.md]: This point is for 1395, which is not yet planned for release

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Overview of Google Business Messages Inbound flow
+
+- This workflow will be triggered for every inbound customer message from the Google Business Messages Asset.
+- This flow contains the basic steps to resolve a conversation and queuing it, when a message is received on the Google Business Messages Asset 
+    ![GBMInboundFlow](../../../images/GBMInboundFlow.jpeg)
+- Google Business Messages can be used to send Text Messages, Images, Rich Cards, and Carousel Cards to your end customers.
+## Inbound flow with suggestions
+
+- This workflow will be triggered for every inbound customer message from the Google Business Messages Asset.
+- When a message is received on the Google Business Messages Asset the flow will offer a few text suggestions on the type of service offered and get the customer response before queuing the contact to live agent for further deliberation
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: queuing the contact to live agent for further handling

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| gbm.agentId  | Unique Identifier for chat asset  |
+| gbm.requestId  | Unique identifier of the message.  |
+  - In the case `gbm.displayName` variable comes empty to the Resolve Node during processing customer name will be defaulted to the value of the flow variable `customerName`
+   ![GBMFlowDisplayName](../../../images/GBMFlowDisplayName.jpeg)
+
+
+#### To be Noted 
+- In the first phase of GBM integration with WxCC only `Incoming Message` is supported from the GBM start node 
+  ![GBMStartNode](../../../images/GBMStartNode.png)
+- The `gbm.conversationId` variables from gbm specific nodes are different from the conversationId variable from Search Conversation response and cannot be used interchangeably
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: Add detail on what is the conversation id from search conversation

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+|Variable Name  | Usage/Descripton  |
+| ------------- | ------------- |
+| gbm.text  | Contains the incoming message from customer  |
+| gbm.timestamp  | Time at  which message was sent   |
+| gbm.conversationId  | Unique Identifier for a chat/conversation with one customer  |
+| gbm.displayName  | Customer name as set in Google account  |
+| gbm.agentId  | Unique Identifier for chat asset  |
+| gbm.requestId  | Unique identifier of the message.  |
+  - In the case `gbm.displayName` variable comes empty to the Resolve Node during processing customer name will be defaulted to the value of the flow variable `customerName`
+   ![GBMFlowDisplayName](../../../images/GBMFlowDisplayName.jpeg)
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: Have a proper customer name or some "Default Customer" value used in flow variable

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+Note : 
+- Queue selection from queue task node is mandatory for any channel flows before making the flow live.
+- Flow variables must be updated in the setting section with the correct asset details before making the flow live.
+
+## Updating Flow Variables
+
+Flow Variables that need to be updated under Custom Variables for different channels: -
+- LiveChat :- appId and liveChatDomain
+
+  <br><img width="800" alt="Settings" src="../../images/Settings.png"><br>
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md]: all images in this page are not rendering properly when we open in MD

File: Webex Connect Flows/Version History.md (hunk 2 above)
Comment (neasaw) [reply Webex Connect Flows/Version History.md]: removed

File: Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD (hunk 1 above)
Comment (neasaw) [reply Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD]: done

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md (hunk 3 above)
Comment (neasaw) [reply Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: done

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md (hunk 5 above)
Comment (neasaw) [reply Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: Done

File: Webex Connect Flows/Version History.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+#Version History
+
+##v3.4
+####Features released:
```
Comment (siramana) [inline Webex Connect Flows/Version History.md]: ```## -> not rendered properly, can you check this file in MD```

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  - Basic Inbound flow
+  - Inbound flow with suggestions
+- Inorder to publish the flows you will need assets configured for Google Business Messages channel you can find the documentation for asset configuration in https://help.imiconnect.io/docs/google-business-messages
+
+## Overview of Google Business Messages Inbound flow
+
+- This workflow will be triggered for every inbound customer message from the Google Business Messages Asset.
+- This flow contains the basic steps to resolve a conversation and queuing it, when a message is received on the Google Business Messages Asset 
+    ![GBMInboundFlow](../../../images/GBMInboundFlow.jpeg)
+- Google Business Messages can be used to send Text Messages, Images, Rich Cards, and Carousel Cards to your end customers.
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: GBM node

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+| gbm.agentId  | Unique Identifier for chat asset  |
+| gbm.requestId  | Unique identifier of the message.  |
+  - In the case `gbm.displayName` variable comes empty to the Resolve Node during processing customer name will be defaulted to the value of the flow variable `customerName`
+   ![GBMFlowDisplayName](../../../images/GBMFlowDisplayName.jpeg)
+
+
+#### To be Noted 
+- In the first phase of GBM integration with WxCC only `Incoming Message` is supported from the GBM start node 
+  ![GBMStartNode](../../../images/GBMStartNode.png)
+- The `gbm.conversationId` variables from gbm specific nodes are different from the conversationId variable from Search Conversation response and cannot be used interchangeably. `gbm.conversationId` is a UUID value while the Search Conversation Response is a string value
```
Comment (siramana) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: Should we provide like below, your thoughts?
gbm.conversationId is google's unique identifier for a customer and flow.conversationId is a WxCC(webex engage)'s unique identifier for a customer like it is for other channels.

File: Webex Connect Flows/Version History.md (hunk 7 above)
Comment (neasaw) [reply Webex Connect Flows/Version History.md]: Fixed

**Review by siramana — DISMISSED** [review]
We will merge during release.

File: Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD [hunk 10]
```diff
@@ -0,0 +1,10 @@
+
+# How to manually upgrade from v3.3 to v3.4 workflows
+
+In case you have made custom changes to your inbound flows, you can manually upgrade your existing v3.3 flows to v3.4. This ensures that the custom changes are retained and don't have to be repeated. Follow the below steps for the same:
+
+## For Google Business Messages 
+- Two new flows have been added in [Google Business Messages](v3.4/Template/Media%20Specific%20Workflows/Google%20Business%20Messages) as references to create flows with Google Business Message Channel 
+
+## For all other flows
```
Comment (prsetiya) [inline Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD:9]: Do we need to mention this if we don't have any change there.

**Review by prsetiya — DISMISSED** [review]

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
+# Overview of Google Business Messages template flows
+
+- This document covers the purpose and configuration details for the below template flows for Google Business Messages channel
+  - Basic Inbound flow
+  - Inbound flow with suggestions
+- Inorder to publish the flows you will need assets configured for Google Business Messages channel you can find the documentation for asset configuration in https://help.imiconnect.io/docs/google-business-messages
+
+## Overview of Google Business Messages Inbound flow
```
Comment (shreemanigandan) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md]: Can we call this as the basic inbound flow? Like we call the next one as Inbound flow with suggestions. Or vice versa to maintain consistency.

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+* **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message over the configured customer chat widget and the widget doesn't have a live chat form configured.
+* **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms sent over the integrated SMS phone number.
+* **Email Inbound Flow** - This workflow will be triggered for every inbound customer email to the integrated email account.
+* **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated whatsapp business number.
+* **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+* **GBM Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured Google Business Messages Agent.
+* **GBM Inbound Flow With Suggestion** - This workflow will be triggered for every inbound customer message over the configured Google Business Messages Agent and will offer the customer a few suggestions.
```
Comment (shreemanigandan) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md]: Apple is missing here, can you please check

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+Note : 
+- Queue selection from queue task node is mandatory for any channel flows before making the flow live.
+- Flow variables must be updated in the setting section with the correct asset details before making the flow live.
```
Comment (shreemanigandan) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md]: >Flow variables must be updated in the setting section with the correct asset details before making the flow live.

Can we please check this? Should it be like
Flow variables must be updated in the setting section with the correct details before making the flow live.

File: Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+
+## Updating Flow Variables
+
+Flow Variables that need to be updated under Custom Variables for different channels: -
```
Comment (shreemanigandan) [inline Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md]: We are not calling gbm or amb here, is this file retained from previuos versions ?

**Pull request closed without merging**